features/bevragen/persoon/adressering/adres-regels/fields.feature

```diff
@@ -0,0 +1,282 @@
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000243                       |
+    | fields              | adressering.adresregel1         |
+    Dan heeft de response een persoon met alleen de volgende 'adressering' gegevens
+    | naam        | waarde                    |
+    | adresregel1 | St. Jacobsstraat 400 LToe |
+
+  Scenario: persoon heeft een adres met straatnaam, huisnummer, huisletter en huisnummertoevoeging die begint met een cijfer als verblijfplaats
+    Gegeven de persoon met burgerservicenummer '000000255' heeft de volgende 'verblijfplaats' gegevens
+    | functie adres (10.10) |
+    | W                     |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | gemeentecode (92.10) | straatnaam (11.10) | huisnummer (11.20) | huisletter (11.30) | huisnummertoevoeging (11.40) |
+    | 0518                 | Cronus             | 555                | B                  | 73c                          |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000255                       |
+    | fields              | adressering.adresregel1         |
+    Dan heeft de response een persoon met alleen de volgende 'adressering' gegevens
+    | naam        | waarde          |
+    | adresregel1 | Cronus 555 B73c |
+
+Rule: adresregel1 wordt voor een adres met straatnaam, huisnummer en aanduiding bij huisnummer 'by' als volgt samengesteld: [straatnaam][spatie][huisnummer] bij [aanduiding bij huisnummer]'
+
+  Scenario: persoon heeft een adres met straatnaam, huisnummer en aanduiding bij huisnummer 'by' als verblijfplaats
+    Gegeven de persoon met burgerservicenummer '000000267' heeft de volgende 'verblijfplaats' gegevens
+    | functie adres (10.10) |
+    | W                     |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | gemeentecode (92.10) | straatnaam (11.10) | huisnummer (11.20) | aanduiding bij huisnummer (11.50) |
+    | 0518                 | Graan voor Visch   | 15201              | by                                |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000267                       |
+    | fields              | adressering.adresregel1         |
+    Dan heeft de response een persoon met alleen de volgende 'adressering' gegevens
+    | naam        | waarde                     |
+    | adresregel1 | Graan voor Visch bij 15201 |
+
+Rule: adresregel1 wordt voor een adres met straatnaam, huisnummer en aanduiding bij huisnummer 'to' als volgt samengesteld: [straatnaam][spatie][huisnummer] t/o [aanduiding bij huisnummer]'
```

Comment:
Wat is de regel voor een adres met straatnaam, huisnummer, huisletter en aanduiding bij huisnummer?
Wat is de regel voor een adres met straatnaam, huisnummer, huisnummertoevoeging die begint met een cijfer en aanduiding bij huisnummer?
Enz?

Reply:
De regel voor samenstelling adresregel1 bij aanduiding bij huisnummer 'to' zou moeten zijn:
[straatnaam][spatie]t/o[spatie][huisnummer]